requirements.txt

```diff
@@ -49,6 +49,9 @@ git+git://github.com/PyCQA/pylint.git@2561f539d60a3563d6507e7a22e226fb10b58210
 # the tests.  With this here, you can run the tests with `py.test` from the base directory.
 pytest
 
+# Static type checking
+mypy
```

Comment:
Can't we do this when we release on pypi though? Then `requirements.txt` can be for installing allennlp for development and we can just specify the actual required packages for the library in `install_requires` in `setup.py`. Just saves us from splitting everything twice, but not too bothered.